### PR TITLE
Updated installation guidelines with more details

### PR DESCRIPTION
**Pull request by shtripat**
Added installation steps for virtualenv and added
more details for configuration changes

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @nthomas-redhat @nnDarshan @anmolbabu @brainfunked please review.

File: doc/source/installation.rst [hunk 1]
```diff
@@ -6,7 +6,27 @@ Installation of latest dev version from the source code::
 
     $ git clone https://github.com/Tendrl/bridge_common.git
     $ cd bridge_common
+    $ yum install http://dl.fedoraproject.org/pub/epel/7/x86_64/p/python-pip-7.1.0-1.el7.noarch.rpm
```
Comment (r0h4n) [inline doc/source/installation.rst]: I dont think we should have distro specific details in install files. 

Either make install files for centos, fedora, ubuntu, or add all those details here

**Comment by shtripat** [comment]
@r0h4n incorporated review comments. Please review..

**Comment by shtripat** [comment]
@brainfunked @r0h4n @nthomas-redhat please review and approve

**Pull request merged**